### DISCUSSION TITLE
Have a question regarding the Camera Flashcarts? ask here

### DISCUSSION THREAD

**Discussion by HDR** [HDR/Gameboy-Camera-Flashcart#1] [General]
If anyone has any questions regarding the Camera Flashcarts you can ask here!

**Comment by jkotrub** [comment]
Is C1 in the flash cart the same capacitor as C1 on the original board? Could it be harvested too? Also what is the value for D1(D2 original) if I need to order one? could D1 from the original board serve if D2 is damaged/lost? Thanks!

**Reply by HDR** [reply]
You can use the original C1 if you want to, i don't have a part number for a replacement Diode unfortunately.

**Comment by 0dysseusRex** [comment]
Would you happen to have a CPL file? I'm looking at ordering a run of boards from JLCPCB since I have a coupon, and I'm probably going to need a placement list.
